config/rbac/role.yaml

```diff
@@ -116,33 +167,40 @@ rules:
   - update
   - watch
 - apiGroups:
-  - anywhere.eks.amazonaws.com
+  - apps
   resources:
-  - oidcconfigs
+  - daemonsets
+  - daemonsets/status
+  - deployments
   verbs:
-  - create
-  - delete
   - get
   - list
-  - patch
-  - update
   - watch
 - apiGroups:
-  - anywhere.eks.amazonaws.com
+  - bmc.tinkerbell.org
   resources:
-  - snowdatacenterconfigs
+  - machines
+  - machines/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - bootstrap.cluster.x-k8s.io
+  resources:
+  - kubeadmconfigtemplates
+  - kubeadmconfigtemplates/status
```

Comment:
why are we writing to the status?
Also, shouldn't we be able to have this only in the eks-a namespace?